javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,193 @@
+(function () {
+  "use strict";
+  // ###### 1.1. Doubling of number
+
+  let numbers = [1, 2, 3, 4];
+  let newNumbers = [];
+
+  numbers.forEach((i) => {
+    if (i % 2 !== 0) {
+      newNumbers.push(i * 2);
+    }
+  });
+
+  // console.log("The doubled numbers are", newNumbers); // [2, 6]
+
+  // ###### 1.2. Working with movies
+
+  // STEP: 1 & 2
+  document
+    .querySelector("button")
+    .addEventListener("click", function getOption() {
+      // declaring variables for getting values for length of the title
+      const minTitleLength = document.querySelector("#min-title-length").value;
+      const maxTitleLength = document.querySelector("#max-title-length").value;
+      // new arrays for title lengths
+      function creatingTitlelength() {
+        if (minTitleLength !== "" && maxTitleLength !== "") {
+          const warning = document.createElement("h2");
+          warning.innerHTML = "Please only fill one option";
+          document.body.appendChild(warning);
+        }
+      }
+      creatingTitlelength();
+      const titleLength = movies.filter((i) => {
```

Comment:
You could name the parameter as "movie" (in singular) instead of i. This might give more information and it also makes the code more readable by clearly stating what the individual elements being filtered are.

Reply:
Thank you for your feedback, i did the changes